### PR TITLE
Add eswikiquote reverted models and revisions for review

### PR DESCRIPTION
**Pull request by Ladsgroup**
Made with ❤️ in stat1005 machine.

Bug: T177762

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1096,6 +1096,80 @@ eswikibooks_tuning_reports: \
...
+datasets/eswikiquote.revisions_for_review.5k_2017.json: \
+		datasets/eswikiquote.autolabeled_revisions.12k_2017.json
+	( \
+	 cat $< | \
+	 grep '"needs_review": true' | \
+	 shuf -n 2500; \
+	 cat $< | \
+	 grep '"needs_review": false' | \
+	 shuf -n 2500 \
+	) | shuf > $@
```
Comment (halfak) [inline Makefile]: We should not use this pattern anymore.  It is out-dated.

**Comment by halfak** [comment]
Just one note on no longer using balanced "needs_review" vs. not datasets

File: Makefile (hunk 1 above)
Comment (Ladsgroup) [reply Makefile]: hmm, what is the new strategy? Can you point out to a wiki that uses it?

Comment (Ladsgroup) [reply Makefile]: Done, looked at old PRs. Hope that's what you mean.

**Pull request merged**